vortexasdk/endpoints/canal_transit_timeseries.py

```diff
@@ -0,0 +1,229 @@
+from datetime import datetime, timedelta
+from typing import Any, Dict, List, Union
+from vortexasdk.api import ID
+
+
+from vortexasdk.api.shared_types import to_ISODate
+from vortexasdk.endpoints.endpoints import CANAL_TRANSIT_TIME_SERIES
+from vortexasdk.endpoints.timeseries_result import TimeSeriesResult
+from vortexasdk.logger import get_logger
+from vortexasdk.utils import convert_to_list
+from vortexasdk.operations import Search
+
+logger = get_logger(__name__)
+
+
+class CanalTransitTimeseries(Search):
+    def __init__(self):
+        Search.__init__(self, CANAL_TRANSIT_TIME_SERIES)
+
+    def search(
+        self,
+        filter_vessel_cubic_capacity_min: int = None,
```

Comment:
All of these types need to be set to Optional if they are being initialised as None